packages/block-library/src/cover/index.php

```diff
@@ -60,11 +60,13 @@ class="wp-block-cover__image-background"
 			esc_attr( $object_position )
 		);
 
-		$content = str_replace(
-			'</span><div',
-			'</span>' . $image . '<div',
-			$content
-		);
+		/* Inserts the featured image between the (1st) cover 'background' `span` and 'inner_container' `div`,
+		   and removes eventual withespace characters between the two (typically introduced at template level) */
+		$inner_container_start = /<div\b[^>]+wp-block-cover__inner-container[\s|"][^>]*>/U;
```

Comment:
@navigatrum this change broke the unit tests. It's a syntax error, regexps in PHP must be enclosed in quotes like this:

```suggestion
		$inner_container_start = '/<div\b[^>]+wp-block-cover__inner-container[\s|"][^>]*>/U';
```

Reply:
I went ahead and committed that suggestion to see if it's going to fix all the tests 🤞